content/basic-concepts.md

```diff
@@ -4,9 +4,9 @@ metaTitle: "sensenet API - Basic concepts"
 metaDescription: "Basic concepts behind sensenet API"
 ---
 
-# Inroduction to sensenet REST API
+# Introduction to sensenet REST API
 
-sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
+Sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
```

Comment:
```suggestion
sensenet offers you a rich API allowing you to manage, integrate and deliver content on all types of clients, devices and platforms. This section explains the API structure and main concepts in more depth.
```
when it is used as the name of the product it is 'sensenet' with a lowercase 's' :)